### ISSUE TITLE
Concrete Strength Detection Using DL

### ISSUE THREAD

**Issue by aindree-2005** [abhisheks008/DL-Simplified#380]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : </br>
Concrete Strength Detection Using DL
:red_circle: **Aim** : </br>
This data set aims to provide a start up for the ones who just started off with deep learning and act as a benchmark.
:red_circle: **Dataset** : </br>
https://www.kaggle.com/datasets/prathamtripathi/regression-with-neural-networking/data
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.
Comparing XGboost vs LightGBM performance on this task
**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
What are the deep learning models you are using for this project?
@aindree-2005

**Comment by aindree-2005** [comment]
> What are the deep learning models you are using for this project? @aindree-2005

My bad! I missed out that part. I was thinking I could use ANNS for this and then compare the effectiveness to XGBoost or LightGBM and make a report on that?

**Comment by abhisheks008** [comment]
Sure. Go ahead. Issue assigned to you @aindree-2005

**Comment by aindree-2005** [comment]
@abhisheks008 made Pr #381

**Comment by abhisheks008** [comment]
Requested some changes.
